orbit/cmd/orbit/orbit.go

```diff
@@ -644,7 +644,7 @@ func main() {
 			log.Info().Err(err).Msg("failed to remove temporary osquery db")
 		}
 
-		if oi.HardwareUUID != orbitHostInfo.HardwareUUID {
+		if runtime.GOOS == "darwin" && oi.HardwareUUID != orbitHostInfo.HardwareUUID {
```

Comment:
We also shouldn't (needn't) run the tmpDBPath stuff from 637 on linux hosts.

Reply:
Done